src/Crypto.gren

```diff
@@ -1337,80 +1366,80 @@ type ImportEcKeyError
-        (extractableToBool extractable)
+        True
         [ "verify" ]
 
 
 {-|-}
-importEcdsaPublicKeyFromSpki : EcNamedCurve -> Bytes -> Extractable -> SecureContext -> Task ImportEcKeyError (PublicKey EcdsaKey EcKeyParams)
-importEcdsaPublicKeyFromSpki namedCurve bytes extractable _context =
+importEcdsaPublicKeyFromSpki : EcNamedCurve -> Bytes -> SecureContext -> Task ImportEcKeyError (PublicKey EcdsaKey EcKeyParams)
+importEcdsaPublicKeyFromSpki namedCurve bytes _context =
     Gren.Kernel.Crypto.importEcKey
         "public"
         "spki"
         bytes
         "ECDSA"
         (ecNamedCurveToString namedCurve)
-        (extractableToBool extractable)
+        True
         [ "verify" ]
 
 
 {-|-}
-importEcdsaPublicKeyFromJwk : EcNamedCurve -> Json.Encode.Value -> Extractable -> SecureContext -> Task ImportEcKeyError (PublicKey EcdsaKey EcKeyParams)
-importEcdsaPublicKeyFromJwk namedCurve jwk extractable _context =
+importEcdsaPublicKeyFromJwk : EcNamedCurve -> Json.Encode.Value -> SecureContext -> Task ImportEcKeyError (PublicKey EcdsaKey EcKeyParams)
+importEcdsaPublicKeyFromJwk namedCurve jwk _context =
     Gren.Kernel.Crypto.importEcKey
         "public"
         "jwk"
         (Gren.Kernel.Json.unwrap jwk)
         "ECDSA"
         (ecNamedCurveToString namedCurve)
-        (extractableToBool extractable)
+        True
         [ "verify" ]
 
 
 {-|-}
-importEcdhPublicKeyFromRaw : EcNamedCurve -> Bytes -> Extractable -> SecureContext -> Task ImportEcKeyError (PublicKey EcdhKey EcKeyParams)
-importEcdhPublicKeyFromRaw namedCurve bytes extractable _context =
+importEcdhPublicKeyFromRaw : EcNamedCurve -> Bytes -> SecureContext -> Task ImportEcKeyError (PublicKey EcdhKey EcKeyParams)
+importEcdhPublicKeyFromRaw namedCurve bytes _context =
     Gren.Kernel.Crypto.importEcKey
         "public"
         "raw"
         bytes
         "ECDH"
         (ecNamedCurveToString namedCurve)
-        (extractableToBool extractable)
+        True
         []
 
 
 {-|-}
-importEcdhPublicKeyFromSpki : EcNamedCurve -> Bytes -> Extractable -> SecureContext -> Task ImportEcKeyError (PublicKey EcdhKey EcKeyParams)
-importEcdhPublicKeyFromSpki namedCurve bytes extractable _context =
+importEcdhPublicKeyFromSpki : EcNamedCurve -> Bytes -> SecureContext -> Task ImportEcKeyError (PublicKey EcdhKey EcKeyParams)
+importEcdhPublicKeyFromSpki namedCurve bytes _context =
     Gren.Kernel.Crypto.importEcKey
         "public"
         "spki"
         bytes
         "ECDH"
         (ecNamedCurveToString namedCurve)
-        (extractableToBool extractable)
+        True
         []
 
 
 {-|-}
-importEcdhPublicKeyFromJwk : EcNamedCurve -> Json.Encode.Value -> Extractable -> SecureContext -> Task ImportEcKeyError (PublicKey EcdhKey EcKeyParams)
-importEcdhPublicKeyFromJwk namedCurve jwk extractable _context =
+importEcdhPublicKeyFromJwk : EcNamedCurve -> Json.Encode.Value -> SecureContext -> Task ImportEcKeyError (PublicKey EcdhKey EcKeyParams)
```

Comment:
Have we discussed the argument order with regards to `SecureContext` before?

`SecureContext` acts a permission value, and we tend to have that as the first parameter in other API's.

Do you remember the rationale for setting it last here?